geopandas/base.py

```diff
@@ -94,7 +94,31 @@ class GeoPandasBase(object):
     @property
     def area(self):
         """Returns a ``Series`` containing the area of each geometry in the
-        ``GeoSeries``."""
+        ``GeoSeries``.
```

Comment:
```suggestion
        ``GeoSeries`` in the CRS of the ``GeoSeries``.
        
        NOTE: this area may be invalid for a geographic CRS; use ``to_crs`` to project geometries to a planar CRS before using this function.
```

Reply:
To be pedantic: the are is not really "in the CRS"? (it also sounds a bit strange to me) But rather "expressed in the unit of the CRS" ?

Reply:
Pedantic & precise is good here!

I think units should be plural?  So `expressed in the units of the CRS`.

Reply:
Isn't there only a single unit? So either meter or feet or degree or ..

Reply:
I'd say that "units" is correct because you'll have 20 meter**s**, hence 20 units. Anyway, this will go to the related PR focusing on texts.